config/v1/feature_gates.go

```diff
@@ -20,9 +20,10 @@ type FeatureGateDescription struct {
 type ClusterProfileName string
 
 var (
-	Hypershift         = ClusterProfileName("include.release.openshift.io/ibm-cloud-managed")
-	SelfManaged        = ClusterProfileName("include.release.openshift.io/self-managed-high-availability")
-	AllClusterProfiles = []ClusterProfileName{Hypershift, SelfManaged}
+	Hypershift            = ClusterProfileName("include.release.openshift.io/ibm-cloud-managed")
```

Comment:
I wonder if we want to create an `IBMCloudManaged` and make this an alias to that, with a view to deprecating and removing this version of `HyperShift` over time